docker/builder/cmd.sh

```diff
@@ -58,7 +58,7 @@ for SUITE_ITERATION in $(seq 1 "${MAX_TEST_SUITE_RETRIES}"); do
 
       set +e
       daemon -U --name sauce_connect -- ${SC_BINARY} \
-        -D *.ciscospark.com,*.wbx2.com,*.webex.com*,storage101.dfw1.clouddrive.com \
+        -D *.ciscospark.com,*.wbx2.com*,*.webex.com*,storage101.dfw1.clouddrive.com \
```

Comment:
Thats correct, I dont want ``` http://internal-testing-services.wbx2.com:8084/api/v1 ``` to go through the tunnel. Isn't that what is intended?